contrib/prometheus-exporter/runner/runner.py

```
@@ -0,0 +1,267 @@
+                LOG.info("Removed stale result file: %s", filename)
+            except OSError as exc:
+                LOG.warning("Could not remove stale file %s: %s", filename, exc)
+
+
+def discover_and_test_models() -> None:
+    """Discover KServe InferenceService models and run load tests."""
+    try:
+        model_pods = v1.list_pod_for_all_namespaces(
+            label_selector="serving.kserve.io/inferenceservice"
+        )
+    except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)